redundant-bmc/src/role_determination.cpp

```diff
@@ -42,6 +42,18 @@ RoleInfo run(const Input& input)
         return {Role::Passive, ErrorCase::noError};
     }
 
+    if (input.previousRole == Role::Active)
+    {
+        lg2::info("Role = active due to that was the previous role");
+        return {Role::Active, ErrorCase::noError};
+    }
+
+    if (input.previousRole == Role::Passive)
```

Comment:
Since if the if condition is true it always returns, the else isn't necessary.  I guess you think it's more readable with the else if, so I can change it.

Reply:
done.